### PR TITLE
Add dladm_vlan module

### PR DESCRIPTION
**Pull request by xen0l**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME
dladm_vlan

##### ANSIBLE VERSION
```
2.3
```

##### SUMMARY
Module to manage VLAN interfaces on illumos/Solaris systems.

### CODE REVIEW

File: lib/ansible/modules/network/illumos/dladm_vlan.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        result['changed'] = True
+
+    if out:
+        result['stdout'] = out
+    if err:
+        result['stderr'] = err
+
+    module.exit_json(**result)
+
+from ansible.module_utils.basic import *
```
Comment (privateip) [inline lib/ansible/modules/network/illumos/dladm_vlan.py]: please use explicit import statements

File: lib/ansible/modules/network/illumos/dladm_vlan.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+main()
```
Comment (privateip) [inline lib/ansible/modules/network/illumos/dladm_vlan.py]: please wrap in `if` block

**Review by privateip — CHANGES_REQUESTED** [review]
* missing RETURN doc string
* missing METADATA
* please consider adding unit test cases

**Comment by xen0l** [comment]
@privateip I addressed you comments. As for unit tests, they will be shared among all modules to a certain degree (I am also speaking about modules, which are already included with Ansible). For that reason, I would like to add them in the next round if possible.

**Comment by xen0l** [comment]
ready_for_review

**Pull request merged**